Update DLRS tutorial with instructions for using Transformers (#1111)
clearlinux/clear-linux-documentation: source/guides/stacks/dlrs.rst

      # Deploy to your cluster.
      ks apply default

Using Transformers* for Natural Language Processing
***************************************************

The DLRS v5.0 release includes `Transformers`_, a state-of-the-art Natural Language Processing (NLP) library for TensorFlow 2.0 and PyTorch. The library is configured to work within the container environment.

> In this section we will using Jupyter Notebook from inside the container to walk through one of the notebooks shown in the `Transformers`_ repository.

Reviewer: ```suggestion
In this section we use a Jupyter Notebook from inside the container to walk through one of the notebooks shown in the `Transformers`_ repository.
```

To run the notebook, you will need to run the Deep Learning Reference Stack, mount it to disk and connect a Jupyter Notebook port.

#. Run the DLRS image with Docker:

   .. code-block:: bash

>       docker run -it -v ${PWD}:/workspace -p 8888:8888 clearlinux/stacks-dlrs-oss

Author: yep :)

>       docker run -it -v ${PWD}:/workspace -p 8888:8888 clearlinux/stacks-dlrs-oss

Reviewer: 👍

#. From within the container, navigate to the workspace, and clone the transformers repository in the container:

  .. code-block:: bash

      cd workspace
      git clone https://github.com/huggingface/transformers.git

#.  Navigate to the Transformers notebook directory, and start a Jupyter Notebook that is linked to the exterior port.  Be sure to copy the token from the output of starting  Jupyter Notebook.

   .. code-block:: bash

      cd transformers/notebooks
      jupyter notebook --ip 0.0.0.0 --no-browser --allow-root

> #. Open a browser and navigate to `localhost:8888`.  If the notebook asks for a token, paste the token from the previous step and submit.  The notebook will also be available at the URL of the system serving the notebook.  For example if you are running on 192.168.1.10, you will be able to access the notebook from other systems on that subnet by navigating to http://192.168.1.10:8888

Reviewer: Ln 553: I tried using `localhost:8888`, but it wasn't clear where to find the `token` from the Terminal output. 
Instead of providing a couple of ways to do this, I recommend using one way that works. (Reword as you like.)

#. Open a browser. 

#. Return to the Terminal where you launched Jupyter Notebook.  Copy one of the URLs  that appears after "Or copy and paste on of these URLs."

#. Paste the URL (with embedded token) into the browser window.

Reviewer: BTW: I forgot to mention it. This is a super cool feature. So nice to open a populated Notebook and understand how this NLP works.
